include/kernel-defaults.mk

```diff
@@ -155,6 +155,24 @@ define Kernel/CompileImage/Default
 	rm -f $(TARGET_DIR)/init
 	+$(KERNEL_MAKE) $(KERNEL_MAKEOPTS_IMAGE) $(if $(KERNELNAME),$(KERNELNAME),all)
 	$(call Kernel/CopyImage)
+
+	# create the Kbuild area
```

Comment:
Upon further experimentation it turned out there needs to be a `rm -rf '$(LINUX_DIR)/module_build'`.  Otherwise it may not recover from an interrupted build.